pom.xml

```diff
@@ -156,6 +156,16 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
```

Comment:
Thanks very much for contributing tests that show the issue and a fix for the issue.

Based on the comment from @timja in https://github.com/jenkinsci/jenkins/pull/5736 (powermock does not seem to work with Java 17 and has much less frequent releases than mockito), would you consider switching to use the mockito framework that is already used in the git plugin, rather than introducing the powermock framework?

I'm not skilled in mocking.  I already struggle to benefit from the mockito based tests and fear that adding another framework will make it more difficult to maintain the plugin.

I removed the powermock framework from the plugin in 2019 with a56d37edbe2c7239b404db760a406328abc39f01 It was used by only one test and was making JDK 11 work more difficult.

Reply:
Got it. I think I am able to make it work with just Mockito. But I need to bring in `mockito-inline`.